android/app/src/main/java/com/shockwallet/NotificationService.java

```diff
@@ -325,6 +336,9 @@ private String EncryptKey(SecretKey key) throws Exception{
     private String DecryptMessage(String response) throws Exception{
         JSONObject resJ = new JSONObject(response);
         String encryptedKey = resJ.getString("encryptedKey");
+        if(encryptedKey.equals("")){
```

Comment:
Please add a comment here

Reply:
This is in case encryption is disabled, if it is, "encryptedKey" will be empty so no need to decrypt the message, will add a comment now